### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by KIsmoilov**
- Created a database named `vet_clinic`.
- Created a table named animals with the following columns: id, name, date_of_birth, escape_attempts, neutered, and weight_kg. 
- Inserted the required data to `schema.sql` file.
- Inserted the required data to `data.sql` file.
- Inserted the required data to `queries.sql` file.

The followings are the screenshots of the results:

Screen Shot 2023-01-17 at 3 13 09 PM

Screen Shot 2023-01-17 at 3 13 15 PM

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon%';
```
Comment (gemmen29) [inline queries.sql]: - [ ] Kindly follow the requirement for the query `Find all animals whose name ends in "mon"`.  Your query will find the animals whose name contains "mon".
### Hint:
```
remove '%' at the end of mon
```

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @KIsmoilov, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by KIsmoilov** [comment]
The following change has been implemented:

- The syntax `%mon%` has been changed to `%mon`.

**Review by OLIPLICHE — APPROVED** [review]
Hi @KIsmoilov , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Implemented the change requested by the previous reviewer 👍 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**